test/wsdls/stockquote.wsdl

```diff
@@ -42,7 +42,7 @@
     </wsdl:portType>
 
     <wsdl:binding name="StockQuoteSoapBinding" type="tns:StockQuotePortType">
-        <soap:binding style="document" transport="http://schemas.xmlsoap.org/soap/http"/>
+        <soap:binding style="rpc" transport="http://schemas.xmlsoap.org/soap/http"/>
```

Comment:
I think the WSDL is from http://www.webservicex.net/stockquote.asmx?WSDL and it's document style. Do you refer to https://github.com/strongloop/loopback-connector-soap/blob/master/test/security.test.js#L40? I cannot tell it's RPC though.